src/api/burn/WixToolset.BootstrapperApplicationApi/IBootstrapperEngine.cs

```diff
@@ -129,14 +129,15 @@ void SetLocalSource(
             );
 
         /// <summary>
-        /// See <see cref="IEngine.SetDownloadSource(string, string, string, string, string)"/>.
+        /// See <see cref="IEngine.SetDownloadSource(string, string, string, string, string, string)"/>.
         /// </summary>
         void SetDownloadSource(
             [MarshalAs(UnmanagedType.LPWStr)] string wzPackageOrContainerId,
             [MarshalAs(UnmanagedType.LPWStr)] string wzPayloadId,
             [MarshalAs(UnmanagedType.LPWStr)] string wzUrl,
             [MarshalAs(UnmanagedType.LPWStr)] string wzUser,
-            [MarshalAs(UnmanagedType.LPWStr)] string wzPassword
+            [MarshalAs(UnmanagedType.LPWStr)] string wzPassword,
+            [MarshalAs(UnmanagedType.LPWStr)] string authorizationHeader
```

Comment:
Nit but we should match the variable name from IEngine.SetDownloadSource(). Is it really `authorizationHeader` or `wzAuthorizationHeader`?

Reply:
It really is authorizationHeader. But none of the other parameter names match...?

Reply:
The `IBootstrapperEngine.cs` is the managed version of `src\api\burn\balutil\inc\IBootstrapperEngine.h` and historically we've kept the Win32/Hungarian names in C# side of the interop files. Here's `SetDownloadSource`:

```cpp
    STDMETHOD(SetDownloadSource)(
        __in_z LPCWSTR wzPackageOrContainerId,
        __in_z_opt LPCWSTR wzPayloadId,
        __in_z LPCWSTR wzUrl,
        __in_z_opt LPCWSTR wzUser,
        __in_z_opt LPCWSTR wzPassword,
        __in_z_opt LPCWSTR wzAuthorizationHeader
        ) = 0;
```